README.md

```diff
@@ -6,136 +6,54 @@
 
 The [Model Context Protocol](https://modelcontextprotocol.io/introduction) (MCP) standardizes how Large Language Models (LLMs) talk to external services like Supabase. It connects AI assistants directly with your Supabase project and allows them to perform tasks like managing tables, fetching config, and querying data. See the [full list of tools](#tools).
 
-## Prerequisites
-
-You will need Node.js ([active LTS](https://nodejs.org/en/about/previous-releases) or newer) installed on your machine. You can check this by running:
-
-```shell
-node -v
-```
-
-If you don't have Node.js 22+ installed, you can download it from [nodejs.org](https://nodejs.org/).
-
 ## Setup
 
-### 1. Personal access token (PAT)
-
-First, go to your [Supabase settings](https://supabase.com/dashboard/account/tokens) and create a personal access token. Give it a name that describes its purpose, like "Cursor MCP Server".
-
-This will be used to authenticate the MCP server with your Supabase account. Make sure to copy the token, as you won't be able to see it again.
-
-### 2. Configure MCP client
-
-Next, configure your MCP client (such as Cursor) to use this server. Most MCP clients store the configuration as JSON in the following format:
+### 1. Follow our security best practices
```

Comment:
Moves this to step 1 to align with docs update in https://github.com/supabase/supabase/pull/36867